packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -933,41 +932,150 @@ class DropdownButtonFormField<T> extends FormField<T> {
   /// The [DropdownButton] [items] parameters must not be null.
   DropdownButtonFormField({
     Key key,
-    T value,
-    @required List<DropdownMenuItem<T>> items,
+    this.value,
+    @required this.items,
+    this.hint,
     this.onChanged,
-    InputDecoration decoration = const InputDecoration(),
+    this.decoration = const InputDecoration(),
     FormFieldSetter<T> onSaved,
     FormFieldValidator<T> validator,
-    Widget hint,
+    bool autovalidate = false,
+    this.disabledHint,
+    this.elevation = 8,
+    this.style,
+    this.icon,
+    this.iconDisabledColor,
+    this.iconEnabledColor,
+    this.iconSize = 24.0,
+    this.isDense = false,
+    this.isExpanded = false,
   }) : assert(decoration != null),
        super(
          key: key,
          onSaved: onSaved,
          initialValue: value,
          validator: validator,
+         autovalidate: autovalidate,
          builder: (FormFieldState<T> field) {
-           final InputDecoration effectiveDecoration = decoration
-             .applyDefaults(Theme.of(field.context).inputDecorationTheme);
+           final InputDecoration effectiveDecoration = decoration.applyDefaults(Theme.of(field.context).inputDecorationTheme);
```

Comment:
We can break this down a little cleaner
```suggestion
           final InputDecoration effectiveDecoration = decoration.applyDefaults(
             Theme.of(field.context).inputDecorationTheme,
           );
```